content/ui-patterns/rails-forms.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Forms in Rails
```

Comment:
Would love to get @emilybrick's input on whether this is the correct spot in the IA for this documentation